e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js

```diff
@@ -100,12 +100,6 @@ context('Insurance - Change your answers - Policy - Broker - Yes to no - As an e
 
         cy.assertEmptyFieldValue(NAME);
         cy.assertEmptyFieldValue(EMAIL);
-
-        // TODO: EMS-3975
-        // cy.checkTextareaValue({
```

Comment:
will this not need to be tested in a future ticket?

Reply:
Yes, when the flow is changed and the check/change answers flow is updated. ~3-5 PRs are required for that.

So just removed it from this test to save having commented code floating around.

Reply:
great, as long as we remember to add the test

Reply:
We will have to, to test changing broker from yes to no and vice versa, once the full flow is in place